src/zenml/orchestrators/base_orchestrator.py

```diff
@@ -208,6 +213,20 @@ def config(self) -> BaseOrchestratorConfig:
         """
         return cast(BaseOrchestratorConfig, self._config)
 
+    @abstractmethod
+    def get_orchestrator_run_id(self) -> str:
```

Comment:
does this even need to be abstract? How about we just define a single env variable that we set for all orchestrators?

Reply:
Environment variables are not the only way to do this, see e.g. non-docker based orchestrators like airflow or the local orchestrator. I also think having it abstract makes it much more visible than silently requiring an environment variable.